Lesson6_2/Citizen.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Lesson6_2
+{
+    class Citizen : Person
```

Comment:
This class should implement IHasId interface and this is the delta between this one and ancestor. ❌